fili-wikipedia-example/Dockerfile

```diff
@@ -26,10 +26,7 @@ RUN mvn -U -B install -DskipTests=true -Dmaven.javadoc.skip=true -Dcheckstyle.sk
 
 EXPOSE 9998
 
-ENTRYPOINT export HOSTIP="$(resolveip -s $HOSTNAME)" \
-&& exec /usr/bin/supervisord -c /etc/supervisor/conf.d/supervisord.conf \
- & mvn -pl fili-generic-example exec:java -Dbard__fili_port=9998 \
-    -Dbard__druid_coord=http://localhost:8081/druid/coordinator/v1 \
-    -Dbard__non_ui_druid_broker=http://localhost:8082/druid/v2 \
-    -Dbard__ui_druid_broker=http://localhost:8082/druid/v2
+COPY entrypoint.sh /entrypoint.sh
```

Comment:
This shouldn't be here since we're cloning the repository in although you will need to change`GITHUB_OWNER=mpardesh` and `BRANCH=filiDocker` while working on it locally (i.e. don't commit that change).

That way when `filiDocker` gets merged into `master` everything should work the same.

You might still have to run `chmod` and you'll need to update the path to `entrypoint.sh` for the `chmod` and `ENTRYPOINT`